src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -291,15 +296,16 @@ struct PrescribedAtmosphere{G, U, P, C, F, R, TP, TI, FT}
     downwelling_radiation :: R
     thermodynamics_parameters :: TP
     times :: TI
-    reference_height :: FT
+    measurement_height :: FT
+    boundary_layer_height :: FT
 end
 
 Base.summary(::PrescribedAtmosphere) = "PrescribedAtmosphere"
 Base.show(io::IO, pa::PrescribedAtmosphere) = print(io, summary(pa))
 
 """
     PrescribedAtmosphere(times;
-                         reference_height,
+                         measurement_height,
```

Comment:
go back to reference height

(this doesn't refer to any measurements, at least not for JRA55 which is our only example)

Reply:
can we call it something different? In COARE 3.6 reference height is something different. Probably `field_height`?

Reply:
What is the reference height in COARE 3.6?

Reply:
I think this like many other parts should have another PR and ideally an issue so we can discuss separately and come up with something well thought out

Reply:
Still wondering what the reference height is in COARE 3.6

Reply:
I couldn't really find it. I think `reference_height` is ok